packages/react-composites/src/composites/ChatComposite/FileDownloadErrorBar.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IIconProps, MessageBar, MessageBarType, Stack } from '@fluentui/react';
+import React from 'react';
+
+/**
+ * @private
+ */
+export interface FileDownloadErrorBarProps {
+  /**callback to dismiss the download error message */
+  onDismissDownloadErrorMessage?: () => void;
+  /** Error message to render */
+  fileDownloadErrorMessage?: string;
```

Comment:
This error should be required (in trivial cases, it is set to `''`)

Reply:
done